test/cmd/newapp.sh

```diff
@@ -118,9 +118,9 @@ os::cmd::expect_success_and_text  'oc new-app -e OPENSHIFT_ENABLE_OAUTH=false -f
 os::cmd::expect_success_and_text 'oc new-app -f test/testdata/template_multiple_resource_gvs.yaml -o yaml' 'apiVersion: apps/v1beta1'
 # check that if an --output-version is requested on a list of varying resource kinds, an error is returned if
 # at least one of the resource groups does not support the given version
-os::cmd::expect_failure_and_text 'oc new-app -f test/testdata/template_multiple_resource_gvs.yaml -o yaml --output-version=v1' 'extensions.Deployment is not suitable for converting'
-os::cmd::expect_failure_and_text 'oc new-app -f test/testdata/template_multiple_resource_gvs.yaml -o yaml --output-version=extensions/v1beta1' 'core.Secret is not suitable for converting'
-os::cmd::expect_failure_and_not_text 'oc new-app -f test/testdata/template_multiple_resource_gvs.yaml -o yaml --output-version=apps/v1beta1' 'extensions.Deployment is not suitable for converting'
+os::cmd::expect_failure_and_text 'oc new-app -f test/testdata/template_multiple_resource_gvs.yaml -o yaml --output-version=v1' 'Deployment is not suitable for converting'
```

Comment:
which group is it in now?

Reply:
still in extensions, just thought to remove it now in case it changes in the future. Can change back if it's better to test groups here

Reply:
Secrets did change to `core`, btw

Reply:
Only update the ones that changed.